NEWS.md

```diff
@@ -1,5 +1,9 @@
 # CHANGES IN GGIR VERSION 2.9-6
 
+- Sleep log: Argument nnigths is now deprecated and number of nights are detected automatically in sleep logs #856
+
+- General: When recorging start after midnight and before 4am, then the wake time for this first time is now derived #859
```

Comment:
You say `general: ` but it only affects part 5 output. So, maybe add at end of sentence: "This will only be used in part 5 output"?

Typo: "recorging" should be "recording"